### PR TITLE
Overload == and != in Room and Hotel

### PR DESCRIPTION
**Pull request by fhrr-sht**

### CODE REVIEW

**Comment by VsIG-official** [comment]
Also, You need to check 'em in [Programm.cs]

**Review by VsIG-official — CHANGES_REQUESTED** [review]
As I said earlier, check this functions in [Program.cs], as We did

**Comment by fhrr-sht** [comment]
на їж

File: Lab4/Project/Program.cs [hunk 1]
```diff
@@ -51,7 +51,7 @@ static void Main()
 			Console.WriteLine("\n--Перевантаження логiчних операторiв:--\n");
 
 			if (RedHotel) Console.WriteLine("Рейтинг додатнiй");
-			RedHotel.Rating = -23;
+			RedHotel.Rating = 23;
```
Comment (VsIG-official) [inline Lab4/Project/Program.cs:54]: And why did You change this field?
You just broke Maxim's test

**Review by VsIG-official — APPROVED** [review]

**Pull request merged**